### ISSUE TITLE
EMA Java - Unable to reset view to get all fields

### ISSUE THREAD

**Issue by bxpw** [Refinitiv/Real-Time-SDK#149]
After sending a ReqMsg with view, the view cannot be removed/reset afterwards by reissuing a blank ReqMsg. Original question is https://community.developers.refinitiv.com/questions/63854/ema-how-do-i-reset-view-to-get-all-fields.html

e.g. after sending
```
long handle = consumer.registerClient(EmaFactory.createReqMsg().payload(view)...);
```
the subsequent request below has no effect. The expected behaviour is that all applicable fields will be received.
```
consumer.reissue(EmaFactory.createReqMsg(), handle);
```
There's another problem with `reissue` that was raised some time ago with issue ID 143, the ticket disappeared. The problem is that `reissue` does not trigger a RefreshMsg to be published, the detailed description is on https://community.developers.refinitiv.com/questions/60045/how-do-i-request-a-refreshmsg-on-an-open-stream.html

Please fix the problem for `ema-3.4.0.1.jar` as well because `ema-3.5.0.1.jar` seems to have a concurrency bug that could cause deadlock in user dispatch mode https://community.developers.refinitiv.com/questions/63662/ema-java-user-dispatcher-thread-deadlock.html

**Comment by APaias** [comment]
@bxpw,  Thank you for bringing this to our attention. A JIRA issue has been created and the issue is being investigated and fix should be available in a future release for the view issue you mention here.

Regarding issue 143: it is still being tracked internally despite it disappearing on GitHub (we are investigating this with GitHub).  When this issue is addressed (it is not yet addressed), it will refer to GitHub #143 in the commit message to indicate a fix. And we will also add a comment here to close it.

Regarding  https://community.developers.refinitiv.com/questions/63662/ema-java-user-dispatcher-thread-deadlock.html, please open a different issue - we are unable to reproduce this issue and prefer to track it separately.

**Comment by bxpw** [comment]
@APaias , thank you for the update. Regarding the deadlock issue, I was asking whether the `reissue` problem could be fixed for `ema-3.4.0.1.jar` because I wouldn't be able to use `ema-3.5.0.1.jar` due to the deadlock problem, I wasn't asking for a fix to the deadlock problem. I haven't raised a ticket for the deadlock issue because I haven't managed to narrow down deadlock code example to a minimal code segment.

**Comment by vlevendel** [comment]
Fix is included with tag, Real-Time-SDK-2.0.1.L1.
